src/overlays/actors/ovl_En_Zl1/z_en_zl1_cutscene_data.c

```diff
@@ -0,0 +1,286 @@
+#include "z_en_zl1.h"
+
+u32 D_80B4C5D0[] = {
+    0x0000001C, 0x00000BB8, 0x0000000A, 0x00000003, 0x00050190, 0x04BB0000, 0xC0000000, 0xFFFFFE5A, 0x00000054,
```

Comment:
same from the other actor, this type is CutsceneData and we may want to generate the macros for them